arangod/Scheduler/SupervisedScheduler.h

```diff
@@ -34,7 +34,11 @@
 #include "Scheduler/Scheduler.h"
 
 namespace arangodb {
-
+#if (_MSC_VER >= 1)
+#define ALIGNAS64
```

Comment:
ooooh :hear_no_evil: 

@dothebart acutally knows that this is a problem - no bashing here - https://arangodb.slack.com/archives/C1PJT16QP/p1560261551221900

Reply:
If we use a macro we could give it a better name like `AVOID_FALSE_SHARING` or say that we want the value of the broken `std::hardware_destructive_interference_size`. It could also be in a common library and/or have a prefix that makes it less likely to be redefined.